test/libyul/yulOptimizerTests/fullSuite/stack_compressor_msize.yul

```diff
@@ -25,21 +30,16 @@
 //
 // {
 //     {
-//         let _1 := gt(not(pc()), 1)
-//         let _2 := pc()
-//         let _3 := not(0)
-//         let _4 := lt(or(1, add(pc(), _3)), 1)
-//         let _5 := pc()
-//         let _6 := pc()
-//         pop(keccak256(pc(), or(gt(not(pc()), 1), 1)))
-//         mstore(lt(or(gt(1, or(or(gt(or(or(or(gt(or(gt(_3, _6), 1), _5), _4), _2), 1), 1), _1), 1)), 1), 1), 1)
-//         sstore(not(pc()), 1)
+//         let _1 := 1
+//         let _2 := gt(not(extcodesize(address())), _1)
```

Comment:
The function is inlined, we have to avoid that. Can you try to make it recursive?

Reply:
I wondered about that one too. I'm trying that one out, yes.